shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,22 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
+  HIGHCONTRAST high_contrast = {.cbSize = sizeof(HIGHCONTRAST)};
+  // API call is only supported on Windows 8+
+  if (SystemParametersInfoW(SPI_GETHIGHCONTRAST, sizeof(HIGHCONTRAST),
+                            &high_contrast, 0)) {
+    BOOL hc_on = high_contrast.dwFlags & HCF_HIGHCONTRASTON;
+    // Currently, only FlutterWindowsView should be used as delegate on Windows
```

Comment:
Consider removing this comment, I'm not sure it adds any value (though let me know if there's something subtle happening here that I missed!)